projects/step-frontend/src/lib/modules/plan/components/plan-list/plan-list.component.ts

```diff
@@ -53,16 +45,16 @@ export class PlanListComponent {
   }
 
   duplicatePlan(id: string): void {
-    this._httpClient
-      .get<any>(`rest/plans/${id}/clone`)
+    this._plansApiService
+      .clonePlan(id)
       .pipe(
-        map((clone) => {
-          clone.attributes.name += '_Copy';
+        map((clone: Plan) => {
+          clone['attributes']!['name']! += '_Copy';
```

Comment:
if we would disable `"noPropertyAccessFromIndexSignature": true` from `tsconfig.workspace.json` we could use the `Plan` type here (even without the explicit `clone: Plan` typing, since `clonePlan()` returns a `Plan` anyways). Just asking: do we use this compilerOption intentionally?

Reply:
@chillbirdo I guess we have it on to be more strict, but for me it would be fine to set it to false. In the very least I would write known property still with the dot syntax ( clone.attributes!['name']! += '_Copy';) If this is the only possible occurence we could also define a type for attributes.